internal/cli/atlas/dbusers/describe.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package dbusers
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/convert"
+	"github.com/mongodb/mongocli/internal/description"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/output"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/mongodb/mongocli/internal/validate"
+	"github.com/spf13/cobra"
+)
+
+type DescribeOpts struct {
+	cli.GlobalOpts
+	store    store.DatabaseUserDescriber
+	authDB   string
+	username string
+}
+
+func (opts *DescribeOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *DescribeOpts) Run() error {
+	r, err := opts.store.DatabaseUser(opts.authDB, opts.ConfigProjectID(), opts.username)
+	if err != nil {
+		return err
+	}
+
+	return output.Print(config.Default(), "", r)
+}
+
+// mongocli atlas dbuser(s) describe <username> --projectId projectId --authDB authDB
+func DescribeBuilder() *cobra.Command {
+	opts := new(DescribeOpts)
+	cmd := &cobra.Command{
+		Use:   "describe <name>",
+		Short: description.DescribeDBUser,
+		Args:  cobra.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			opts.username = args[0]
+
+			validAuthDBs := []string{convert.AdminDB, convert.ExternalAuthDB}
+			if err := validate.FlagInSlice(opts.authDB, flag.AuthDB, validAuthDBs); err != nil {
+				return err
+			}
+
+			return opts.PreRunE(opts.initStore)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return opts.Run()
+		},
+	}
+
+	cmd.Flags().StringVar(&opts.ProjectID, flag.ProjectID, "", usage.ProjectID)
+
+	cmd.Flags().StringVar(&opts.authDB, flag.AuthDB, "", usage.AuthDB)
```

Comment:
[non-blocking] could we default to `admin` here? I think this may also solve the error we have on evergreen